src/analytics.ts

```diff
@@ -95,17 +93,21 @@ export class Analytics {
   }
 
   protected formatBucketAggregate(
-    rawAggregate: [string, number][],
+    rawAggregate: [string | 1 | null, number][],
```

Comment:
Maybe we should use some sort of branded types for those. Right now, I have no idea what that 1 stands for. I'd love to see this:

```
type Todo = 1 | null
rawAggregate: [string | Todo, number][],

```